pkg/datagatherer/istio/istio_test.go

```diff
@@ -0,0 +1,191 @@
+package isito
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"os"
+	"testing"
+
+	"gopkg.in/yaml.v2"
+)
+
+const tempFilePrefix = "preflight-test-istio-datagatherer"
+
+//// TestFetchRealCluster is not used, but has been left here for development use to allow testing against a real cluster.
```

Comment:
I think I'd rather this was stored as a personal snippet since we don't have it for our other datagatherers.